### PR TITLE
Add option to ignore packages if they appear first in a stack frame

### PR DESCRIPTION
**Pull request by cthornton**

### CODE REVIEW

File: create_flamegraph.sh [hunk 1]
```diff
@@ -36,13 +36,13 @@ function help {
...
 jfr_file=""
 
-while getopts "df:airsx:y:e:" opts
+while getopts "df:airsx:y:e:op:" opts
 do
   case $opts in
     f)
         jfr_file=${OPTARG}
         ;;
-    \?)
```
Comment (cthornton) [inline create_flamegraph.sh:45]: I should probably undo this change here

**Comment by chrishantha** [comment]
Hi @cgthornt, Why did you need this change? What is special about the very first package name?

**Comment by cthornton** [comment]
Hi @chrishantha, these changes were for something internal we were doing. Specifically, we wanted to remove stack trace items from a framework (i.e. removing jetty stack traces). We don't need to merge this into the main jfr-flame-graph repo though
